Training24/PrimeNumber.c

```diff
@@ -0,0 +1,78 @@
+// ------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// Dhaarani Mohanasundaram
+// -------------------------------------
+// PrimeNumber.c
+// Program on B4 branch.
+// Program that takes a number as input and displays whether it is a prime number.
+// ------------------------------------------------------------------------------------
+#include <stdio.h>
+#include <stdbool.h>
+#include <stdlib.h>
+
+#define PRIME 1
+#define NOT_PRIME 0
+
+/// <summary>To check if a number is prime.</summary>
+int PrimeChecker (int num);
+
+/// <summary>To run few test cases with PrimeChecker.</summary>
+void TestCases ();
+
+/// <summary>To check PrimeChecker with user input.</summary>
+void CheckPrimeForUserInput ();
+
+int main () {
+   TestCases ();
+   CheckPrimeForUserInput ();
+   return 0;
+}
+
+int PrimeChecker (int num) {
+   if (num <= 1) return NOT_PRIME;
+   if (num == 2) return PRIME;
+   for (int i = 2; i <= num / 2; i++) {
+      if (num % i == 0) return NOT_PRIME;
+   }
+   return PRIME;
+}
+
+void TestCases () {
+   int testNumbers[] = { 113, 15, 2, 1, 29 },
```

Comment:
You are limiting your test check by giving some least of inputs. Instead of that create random array and test